.sage/main.go

```diff
@@ -24,6 +24,13 @@ func main() {
 	)
 }
 
+func DoubleGitVerifyNoDiff(ctx context.Context) error {
+	sg.SerialDeps(ctx, GitVerifyNoDiff)
+	sg.SerialDeps(ctx, FormatMarkdown)
+	sg.SerialDeps(ctx, GitVerifyNoDiff) // this should report a change
```

Comment:
Unexpectedly the second call to `GitVerifyNoDiff` does not return an error, even though the previous format changed a file